cmd/utils.go

```diff
@@ -59,3 +60,32 @@ func preRunConfig(cmd *cobra.Command, args []string) {
 		log.Info("config file not found, proceeding without it")
 	}
 }
+
```

Comment:
I believe all these methods should go to util/file. There is a `ArtifactPath` method which has been implemented before that you can leverage.

Reply:
Understood. However, but I think the methods in util/file are not exposed to be used. I could be wrong.

Reply:
They are exposed as public methods, so you can move stuffs there and reuse them in all checks

Reply:
make build
go build -o preflight -ldflags "-X github.com/redhat-openshift-ecosystem/openshift-preflight/version.commit=d6ea2fbf6472c69584c80ab197297761c714a67a -X github.com/redhat-openshift-ecosystem/openshift-preflight/version.version="0.0.0"" main.go
package command-line-arguments
	imports github.com/redhat-openshift-ecosystem/openshift-preflight/cmd
	cmd/utils.go:9:2: use of internal package github.com/redhat-openshift-ecosystem/openshift-preflight/certification/internal/utils/file not allowed
make: *** [Makefile:10: build] Error 1

https://stackoverflow.com/questions/59342373/use-of-internal-package-not-allowed

Reply:
Perhaps move it to certification/utils/utils.go? Then the internal packages can utilize it, as well as the cmd package.

Reply:
Will do.